src/goto-instrument/rw_set.h

```diff
@@ -87,7 +87,7 @@ class rw_set_baset
   void output(std::ostream &out) const;
 
 protected:
-  virtual void track_deref(const entryt &entry, bool read) {}
+  virtual void track_deref(const entryt &, bool read) { (void)read; }
```

Comment:
I'm probably missing some subtlety here but this one is not like it's friends...